### PR TITLE
mpdas: add service

### PR DESCRIPTION
**Pull request by ixearth**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

mpdas is an alternative to [mpdscribble](https://github.com/Homebrew/homebrew-core/blob/master/Formula/mpdscribble.rb), it should also have a service

### CODE REVIEW

File: Formula/mpdas.rb [hunk 1]
```diff
@@ -21,6 +21,35 @@ def install
     etc.install "mpdasrc.example"
   end
 
+  def caveats; <<~EOS
+    The example configuration file was placed in #{etc}/mpdasrc.example
+  EOS
+  end
```
Comment (zbeekman) [inline Formula/mpdas.rb]: Why did you add this caveat? Caveats should only be for homebrew specific issues/quirks. At the very least the text should be in the present perfect tense ("is placed") because the caveat is printed during `brew info` as well as after installation.

Comment (ixearth) [reply Formula/mpdas.rb]: Because I saw mpdscribble have a caveat that print the path of configuration file, and during configure mpdas I tried to find the configuration file by listing files in the package and noticed mpdasrc.example, thought it could be useful for ppl just installed this package and don't know how to configure it.
But you are right, printing this is not necessary and could be annoying, the example configuration file does not have any comments, ppl can create the configuration file by looking at manpage. I will remove the caveat.

Comment (ixearth) [reply Formula/mpdas.rb]: sorry I'm not familiar with git pull request, the commit message of the second commit does not looks fit.. Can you squash these commit while merging?

**Comment by fxcoudert** [comment]
Thanks @ixearth
